R/build-search-docs.R

```diff
@@ -314,3 +308,13 @@ is_heading <- function(node) {
 capitalise <- function(string) {
   paste0(toupper(substring(string, 1, 1)), substring(string, 2))
 }
+
+get_site_paths <- function(pkg) {
+  paths <- fs::path_rel(
```

Comment:
Can you please split this into multiple independent lines? e.g. `paths <- ...`, `paths_rel <- ...`